cmd/swarmctl/service/flagparser/plugin.go

```diff
@@ -0,0 +1,18 @@
+package flagparser
+
+import (
+	"github.com/docker/swarmkit/api"
+	"github.com/spf13/pflag"
+)
+
+func parsePlugin(flags *pflag.FlagSet, spec *api.ServiceSpec) error {
```

Comment:
It's a consistency thing - in the CLI, every `parse*` function is a `func(flags *pflag.FlagSet, spec *api.ServiceSpec) error`.

Agreed to keep it this way?